### PR TITLE
Message priority & acknowledgements

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: 
- https://github.com/mattermost/mattermost-server/pull/20875
- https://github.com/mattermost/mattermost-webapp/pull/10951

Includes end user workflows, system admin configuration setting details, and telemetry.

### CODE REVIEW

**Review by Rishselva — CHANGES_REQUESTED** [review]
Great work Carrie! 

The only piece of information that could be added is letting the user know that the settings are configurable.

**Comment by cwarnermm** [comment]
Thanks, @Rishselva! Where would someone be able to change that 5 minute setting? Is that a config option we surface up in the System Console for admins?

**Comment by Rishselva** [comment]
Yes, so if the admin goes to System Console > Posts they'll be able to see all the config options. 

Screen Shot 2022-12-09 at 12 47 53 PM

File: source/channels/message-priority.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Set message priority (Beta)
```
Comment (justinegeffen) [inline source/channels/message-priority.rst:1]:
```suggestion
Set message priority (beta)
```

**Review by justinegeffen — APPROVED** [review]

**Review by Rishselva — CHANGES_REQUESTED** [review]
Looks great! 

Since we're cancelling v7.6 we might want to update the doc to say v7.7. Additionally, is there a reason why persistent notification is not included in this document?

**Comment by cwarnermm** [comment]
Thanks, @Rishselva! Yes, I'll update this PR to change references from v7.6 to v7.7, and I'll update this PR to include the persistent notification functionality. It's not there yet because this PR focused on v7.6 functionality only and persistent notifications were scheduled for v7.7 at the time this PR was in flight.

**Comment by Rishselva** [comment]
Hello! Recent update, we will not be including persistent notifications in the v7.7 release. We are okay to leave it out of this doc if its being released with v7.7.

**Comment by cwarnermm** [comment]
Thanks, @Rishselva! I appreciate the update. I'll leave persistent notifications out of this PR. That feature will be documented closer to its scheduled release date.

**Comment by esethna** [comment]
I will leave PM review to @Rishselva

**Review by Rishselva — APPROVED** [review]
Hello! Just reviewed the document and all of the information looks good! Thank you :)

File: source/channels/message-priority.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+.. |priority-icon| image:: ../images/Priority-Message-Icon.svg
+  :alt: Select the Priority icon to mark new messages as important or urgent before sending them.
+
+.. |acknowledge-button| image:: ../images/Ack-Button-Default.svg
+  :alt: Select the Acknowledge button to indicate that you've read it and taken necessary action.
+
+.. |acknowledged-icon| image:: ../images/Ack-Button-Active.svg
+  :alt: Hover over the Acknowledged icon to see who has read and actioned the message. Select the icon to remove your acknowledgement within five minutes.
+
+From Mattermost v7.6, you can add a message priority label to root messages to make important messages requiring timely action or response more visible and less likely to be overlooked.
```
Comment (esethna) [inline source/channels/message-priority.rst:16]: Mattermost v7.7? cc// @Rishselva @cwarnermm

File: source/channels/message-priority.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+.. image:: ../images/Message-Priority-Animated-optimized.gif
+  :width: 700
+  :alt: Ensure important and urgent messages stand out clearly by adding priority labels to root messages.
+
+Set message priority
```
Comment (esethna) [inline source/channels/message-priority.rst:22]: Mobile can do this too, so we  should probably take the new tabbed approach to this content. cc// @cwarnermm @Rishselva

File: source/channels/message-priority.rst [hunk 4]
```diff
@@ -0,0 +1,46 @@
+Set message priority (beta)
```
Comment (esethna) [inline source/channels/message-priority.rst:1]: SInce it's in beta, how do people give feedback (ie the give feedback button in the pop-over)

Comment (esethna) [inline source/channels/message-priority.rst:1]: How do admins disable it if desired?

Comment (esethna) [inline source/channels/message-priority.rst:1]: Can we try to answer the question of "when should I use a message priority label"? ie we don't want people to abuse it. So providing some example use cases for the priority labels and ack might be useful in the docs

**Review by esethna — COMMENTED** [review]
@cwarnermm @Rishselva I provided some suggestions for improvements on the doc page inline

**Pull request merged**